docs/user/security/securing-communications/index.asciidoc

```diff
@@ -3,122 +3,131 @@
+the FQDN.
 
-// TBD: The reference information for server.ssl.enabled says it "enables SSL for 
-// outgoing requests from the Kibana server to the browser". Do we need to 
-// reiterate here that only one side of the communications is encrypted?
+You may choose to generate a certificate signing request (CSR) and private key using the {ref}/certutil.html[`elasticsearch-certutil`] tool.
 
-For more information, see <<settings,{kib} configuration settings>>.
---
+For example:
+[source,sh]
+--------------------------------------------------------------------------------
+bin/elasticsearch-certutil csr -name kibana-server -dns some-website.com,www.some-website.com
+--------------------------------------------------------------------------------
+This will produce a ZIP archive named `kibana-server.zip`. Extract that archive to obtain the PEM-formatted CSR (`kibana-server.csr`) and
+unencrypted private key (`kibana-server.key`). In this example, the CSR has a common name (CN) of `kibana-server`, a SAN of
+`some-website.com`, and another SAN of `www.some-website.com`.
 
-. Configure {kib} to connect to {es} via HTTPS:
-+
+NOTE: You will need to use a certificate authority (CA) to sign your CSR to obtain your server certificate. This certificate's signature
+will be verified by web browsers that are configured to trust the CA.
 --
-NOTE: To perform this step, you must 
-{ref}/configuring-security.html[enable the {es} {security-features}] or you 
-must have a proxy that provides an HTTPS endpoint for {es}. 
 
---
+. Configure {kib} to access the server certificate and private key.
 
-.. Specify the HTTPS protocol in the `elasticsearch.hosts` setting in the {kib}
-configuration file, `kibana.yml`:
+.. If your server certificate and private key are in PEM format:
 +
 --
+Specify your server certificate and private key in `kibana.yml`:
 [source,yaml]
 --------------------------------------------------------------------------------
-elasticsearch.hosts: ["https://<your_elasticsearch_host>.com:9200"]
+server.ssl.certificate: "/path/to/kibana-server.crt"
+server.ssl.key: "/path/to/kibana-server.key"
+--------------------------------------------------------------------------------
+If your private key is encrypted, add the decryption password to your <<secure-settings,{kib} keystore>>:
+[source,yaml]
+--------------------------------------------------------------------------------
+bin/kibana-keystore add server.ssl.keyPassphrase
 --------------------------------------------------------------------------------
 --
-
-.. If you are using your own CA to sign certificates for {es}, set the
-`elasticsearch.ssl.certificateAuthorities` setting in `kibana.yml` to specify
-the location of the PEM file.
+.. Otherwise, if your server certificate and private key are contained in a PKCS#12 file:
 +
 --
+You will need to extract the server certificate and private key in PEM format.
```

Comment:
Doesn't https://github.com/elastic/kibana/pull/53810 mean that users don't _need to_ do this but could also use the PKCS#12 as is ?

Reply:
> Doesn't #53810 mean that users don't _need to_ do this but could also use the PKCS#12 as is ?
Yes but that only works on Kibana 7.6.0+. This PR is to spruce up older documentation.